From 7bbadaec5ab9c60bd5ad8e1feee39af9f170b552 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 14 Aug 2025 14:57:45 +0100
Subject: [PATCH] daemon/inspect_fs_windows.ml: Add debugging when we start
 registry analysis

Add some debugging when we begin the process of analyzing the Windows
registry of a guest.
---
 daemon/inspect_fs_windows.ml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/daemon/inspect_fs_windows.ml b/daemon/inspect_fs_windows.ml
index 5991cdba3..00acf5196 100644
--- a/daemon/inspect_fs_windows.ml
+++ b/daemon/inspect_fs_windows.ml
@@ -207,6 +207,12 @@ and check_windows_registry systemroot data =
     if Is.is_file system_hive then Some system_hive else None in
   data.windows_system_hive <- system_hive;
 
+  if verbose () then
+    eprintf "check_windows_registry: software hive: %s\n\
+             check_windows_registry: system hive: %s\n%!"
+      (Option.value ~default:"None" software_hive)
+      (Option.value ~default:"None" system_hive);
+
   match software_hive, system_hive with
   | None, _ | Some _, None -> ()
   | Some software_hive, Some system_hive ->
-- 
2.47.1

